standalone/instructlab-rhoai.md

```diff
@@ -0,0 +1,363 @@
+# Distributed InstructLab Training on RHOAI
+
+This file documents an experimental install of InstructLab on Red Hat OpenShift AI.
+
+## Pre-requisites
+
+* An OpenShift cluster with 
+    * Sufficient GPU nodes available for training. 
+        * 2 nodes with at least 2 NVIDIA A100 GPUs each.
```

Comment:
Is the number of nodes a pre-requisite? My understanding is what matters is the number of GPUs.

Reply:
I guess the related config info is taken from here : https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_ai/1.1/html/getting_started/hardware_requirements_rhelai#bare-metal
But need to reframe the sentence a bit